doc/API/Bills.md

```diff
@@ -132,6 +132,66 @@ Output:
 }
 ```
 
+### `get_bill_graph`
+
+Retrieve a graph associated with a bill
+
+Route: `/api/v0/bills/:id/graphs/:graph_type
+
+Example:
+
+```curl
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/graphs/bits
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/graphs/bits?from=1517443200
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/graphs/bits?from=1517443200&to=1517788800
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/graphs/monthly
+```
+
+Output:
+
+(image)
+
+### `get_bill_graphdata`
+
+Retrieve the data used to draw a graph so it can be rendered in an external system
+
+Route: `/api/v0/bills/:id/graphdata/:graph_type
+
+Example:
+
+```curl
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/graphdata/bits
```

Comment:
This isn't valid:

```
{
    "status": "error",
    "message": "from parameter must be supplied"
}
```

Reply:
@richardlawley This now returns a http 500.

Reply:
Unable to reproduce this again, on both my dev and live installs.  Do you get a log message, or perhaps when adding &debug to the URL?

Reply:
Your latest commit fixes that now :)

Reply:
I've just found that the bills/1/history/1/graphdata/bits was giving a 500 error - is it possible you meant this route rather than bills/1/graphdata/bits ?

If so, this is now fixed.

Reply:
Possibly yes but it's definitely fixed as I checked out your previous commit and it broke again.